routing/router_test.go

```diff
@@ -992,8 +1004,10 @@ func TestAddEdgeUnknownVertexes(t *testing.T) {
 
 	// Should still be able to find the routes, and the info should be
 	// updated.
-	routes, err = ctx.router.FindRoutes(targetNode, paymentAmt,
-		defaultNumRoutes, DefaultFinalCLTVDelta)
+	routes, err = ctx.router.FindRoutes(
```

Comment:
Could add a short test here for the fee limit exceeded case?

Reply:
Fixed.